caikit_nlp/modules/text_generation/fine_tuning.py

```diff
@@ -0,0 +1,262 @@
+# Copyright The Caikit Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+# Third Party
+from torch.utils.data import IterableDataset
+from transformers import (
+    AutoConfig,
+    AutoTokenizer,
+    DataCollatorForSeq2Seq,
+    Seq2SeqTrainer,
+    Seq2SeqTrainingArguments,
+    Trainer,
+)
+
+# First Party
+from caikit.core.data_model import DataStream
+from caikit.core.modules import ModuleBase, module
+from caikit.core.toolkit import error_handler, wip_decorator
+import alog
+
+# Local
+from ...data_model import GeneratedResult, GenerationTrainRecord
+from ...toolkit.data_stream_wrapper import SimpleIterableStreamWrapper
+from ...toolkit.data_type_utils import get_torch_dtype
+from .text_generation_task import TextGenerationTask
+
+log = alog.use_channel("TXT_GEN")
```

Comment:
```suggestion
log = alog.use_channel("FIN_TUN_GEN")
```

So that we don't have collisions in logging channels with the separate text generation module